MediaBrowser.Providers/Manager/ProviderManager.cs

```diff
@@ -970,9 +970,8 @@ public void OnRefreshProgress(BaseItem item, double progress)
             var id = item.Id;
             _logger.LogInformation("OnRefreshProgress {0} {1}", id.ToString("N", CultureInfo.InvariantCulture), progress);
 
-            if (_activeRefreshes.ContainsKey(id))
+            if (_activeRefreshes.TryAdd(id, progress))
```

Comment:
This doesn't feel correct, it changes the behaviour to the opposite of how it was before.

Pre-change it updated `_activeRefreshes[id]` with `progress`, now it sets it to `progress` _only_ if `id` is missing from `_activeRefreshes`.